dbfit-java/postgres/src/main/java/dbfit/environment/PostgresEnvironment.java

```diff
@@ -74,7 +74,7 @@ protected String parseCommandText(String commandText) {
                     paramName = "";
                 String dataType = rs.getString(2);
                 DbParameterAccessor dbp = createDbParameterAccessor(
-                        paramName,
+                		'"'+paramName+'"',
```

Comment:
Formatting notes: spaces are missing around `+`; and indentation should be spaces.